t_sparse_list.py

```diff
@@ -109,6 +109,17 @@ def test_slice_reversal_empty(self):
         sl = sparse_list.SparseList(4)
         self.assertEquals([None, None, None, None], sl[::-1])
 
+    def test_slice_with_list_read(self):
+        sl = sparse_list.SparseList([1, 2, 3, 4, 5])
+        ip = [0, 2, 4]
```

Comment:
Yes, it's index positions. I could call it like so, but in the moment, a two-character variable seemed appropriate for symmetry. :3